### PR TITLE
Viewmodels remove dapper

### LINKED ISSUE #132: Update "Creating a Student with View Models" Chapter in Book 3
1. Replace Dapper with ado.net

### PR DESCRIPTION
**Pull request by stevebrownlee**
Converted all code from Dapper to ADO.NET

Closes #132

### CODE REVIEW

File: book-3-workforce/chapters/VIEWMODELS.md [hunk 1]
```diff
@@ -51,33 +51,88 @@ namespace StudentExercises.Models.ViewModels
...
-        /*
-            Need to pass along the IConfiguration to get our
-            database connection string
-         */
-        public StudentInstructorViewModel(IConfiguration config)
+        private SqlConnection Connection
+        {
+            get
+            {
+                string _connectionString = "___YOUR CONNNECTION STRING HERE____";
```
Comment (askingalot) [inline book-3-workforce/chapters/VIEWMODELS.md]: I think we should pass in the connection string as a constructor parameter.

File: book-3-workforce/chapters/VIEWMODELS.md [hunk 2]
```diff
@@ -170,25 +226,43 @@ namespace StudentExercises.Models.ViewModels
 
         public StudentCreateViewModel(IConfiguration config)
```
Comment (askingalot) [inline book-3-workforce/chapters/VIEWMODELS.md]: Remove the `config` param and add a connection string param.

File: book-3-workforce/chapters/VIEWMODELS.md (hunk 1 above)
Comment (stevebrownlee) [reply book-3-workforce/chapters/VIEWMODELS.md]: Changed

File: book-3-workforce/chapters/VIEWMODELS.md (hunk 2 above)
Comment (stevebrownlee) [reply book-3-workforce/chapters/VIEWMODELS.md]: Removed

**Pull request merged**